corehq/apps/smsbillables/models.py

```diff
@@ -225,6 +226,11 @@ def get_by_criteria(cls, direction, domain=None):
         return cls.objects.filter(criteria=criteria.id).latest('date_created')
 
 
+@quickcache(['sms_backend_id'])
+def _sms_backend_is_global(sms_backend_id):
+    return SMSBackend.get(sms_backend_id).is_global
```

Comment:
@dannyroberts Yup, that's right, is_global gets set once on backend create and never changes after that.  But it's good to know that if it ever does change in some rare situation that this would pick it up after 10 minutes (which seems fine).

@nickpell fyi for sms billing